go.mod

```diff
@@ -76,7 +76,7 @@ require (
 	github.com/DataDog/gopsutil v0.0.0-20191127151039-7e1a4eadb59e
 	github.com/DataDog/mmh3 v0.0.0-20200316233529-f5b682d8c981 // indirect
 	github.com/DataDog/watermarkpodautoscaler v0.1.0
-	github.com/DataDog/zstd v1.4.5
+	github.com/DataDog/zstd v0.0.0-20160706220725-2bf71ec48360
```

Comment:
FWIW, this appears to pin a commit (`2bf71ec48360`) that's different than what we were pinning in 7.19.0: https://github.com/DataDog/datadog-agent/blob/7.19.0/Gopkg.lock#L130

Reply:
diff appears minimal, and only affects decompression, can probably be ignored: https://github.com/DataDog/zstd/compare/2bf71ec4836011b92dc78df3b9ace6b40e65f7df...2b373cbe6ac0c8e6960bbd18026ceb269eef89f5